doc/development_process/issues.rst

```diff
@@ -9,3 +9,54 @@ a reference to the feature by mentioning the corresponding Issue or pull-request
 
 At any time it should be possible to establish the origin of a change and the
 reason behind it by following the references in the code.
+
+
+Feature Tracking
+*****************
+For feature tracking we use Github labels to classify new features and
+enhancements. The following is the description of each category:
+
+Enhancement
+  Changes to existing features that are not considered a bug and would not a
+  block a release. This is an incremental enhancement to a feature that already
+  exists in Zephyr.
+
+Feature request
+  A request for a feature that is not on the roadmap yet that has not been
+  vetted yet and needs further discussion and details.
+
+Feature
+  A high level roadmap feature with a detailed proposal with an owner. Features
+  require a detailed architectural design and must go through an RFC process.
```

Comment:
"must go through an RFC process."

What would this RFC process be?
A thought: Today, many features are added by the maintainers of an area with only some internal discussion or not even. And for anything non controversial, which just adds new functionality and does not break anything, it would seem perfectly ok.